source/server-selection/server-selection.rst

```diff
@@ -624,9 +652,11 @@ A valid ``$readPreference`` document for mongos has the following requirements:
     The ``maxStalenessSeconds`` field MUST be either be absent or be present
     exactly once with an integer value.
 
+    The ``hedge`` field MUST be either absent or be an object.
+
 Mongos receiving a query with ``$readPreference`` SHOULD validate the
-``mode``, ``tags``, and ``maxStalenessSeconds`` fields according to rules 1 and 2 above,
-but SHOULD ignore unrecognized fields for
+``mode``, ``tags``, ``maxStalenessSeconds``, and ``hedge`` fields according to
+rules 1 and 2 above, but SHOULD ignore unrecognized fields for
 forward-compatibility rather than throwing an error.
```

Comment:
"SHOULD ignore unrecognized fields for forward-compatibility rather than throwing an error" makes me think that mongos' current behavior may be intentional. Perhaps this should be brought up in [SERVER-46093](https://jira.mongodb.org/browse/SERVER-46093) before someone from the server team follows up on it.

Reply:
I left a [comment in SERVER-46093](ttps://jira.mongodb.org/browse/SERVER-46093?focusedCommentId=2877497&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2877497).